get-customer-by-org-number/src/test/java/no/unit/nva/customer/get/GetCustomerByOrgNumberHandlerTest.java

```diff
@@ -0,0 +1,116 @@
+package no.unit.nva.customer.get;
+
+import static no.unit.nva.customer.testing.TestHeaders.getErrorResponseHeaders;
+import static no.unit.nva.customer.testing.TestHeaders.getRequestHeaders;
+import static no.unit.nva.customer.testing.TestHeaders.getResponseHeaders;
+import static nva.commons.handlers.ApiGatewayHandler.ALLOWED_ORIGIN_ENV;
+import static org.apache.http.HttpStatus.SC_BAD_REQUEST;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.zalando.problem.Status.BAD_REQUEST;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.util.Map;
+import java.util.UUID;
+import no.unit.nva.customer.ObjectMapperConfig;
+import no.unit.nva.customer.model.Customer;
+import no.unit.nva.customer.service.CustomerService;
+import no.unit.nva.testutils.HandlerRequestBuilder;
+import nva.commons.handlers.GatewayResponse;
+import nva.commons.utils.Environment;
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.zalando.problem.Problem;
+
+public class GetCustomerByOrgNumberHandlerTest {
+
+    public static final String WILDCARD = "*";
+    public static final String REQUEST_ID = "requestId";
+    public static final String SAMPLE_ORG_NUMBER = "123";
+    public static final String EXPECTED_ERROR_MESSAGE = "Missing from pathParameters: orgNumber";
+
+    private ObjectMapper objectMapper = ObjectMapperConfig.objectMapper;
+    private CustomerService customerServiceMock;
+    private Environment environmentMock;
+    private GetCustomerByOrgNumberHandler handler;
+    private ByteArrayOutputStream outputStream;
+    private Context context;
+
+    /**
+     * Setting up test environment.
+     */
+    @BeforeEach
+    @SuppressWarnings("unchecked")
```

Comment:
Nothing here is unchecked

Reply:
How can you tell? I get nothing in my IDE only warnings during Gradle build (with no line numbers)